crates/re_renderer_examples/depth_cloud.rs

```diff
@@ -128,7 +128,10 @@ impl RenderDepthClouds {
         );
 
         let command_buffer = view_builder
-            .queue_draw(GenericSkyboxDrawData::new(re_ctx))
+            .queue_draw(GenericSkyboxDrawData::new(
+                re_ctx,
+                re_renderer::renderer::GenericSkyboxType::DirectionalGradientDark,
```

Comment:
you could make the examples a bit shorter with

```suggestion
                Default::default(),
```